Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:15.04
+MAINTAINER CDIS <cdissupport@opensciencedatacloud.org>
+
+RUN apt-get update && apt-get install -y python-pip git python-dev postgresql postgresql-client libpq-dev apache2 libapache2-mod-wsgi \ 
```

Comment:
if we want to use postgres I think it will be a dedicated postgres server outside of the docker so there is no need to install any postgres packages here except `libpq-dev`?